.github/workflows/scorecard.yaml

```
@@ -0,0 +1,77 @@
+
+name: Scorecard supply-chain security
+on:
+  # For Branch-Protection check. Only the default branch is supported. See
+  # https://github.com/ossf/scorecard/blob/main/docs/checks.md#branch-protection
+  branch_protection_rule:
+  # To guarantee Maintained check is occasionally updated. See
+  # https://github.com/ossf/scorecard/blob/main/docs/checks.md#maintained
+  schedule:
+    - cron: '45 1 * * 4'
+  push:
+    branches: [ "master" ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)